src/hooks/UrqlUseQuery.rei

```diff
@@ -0,0 +1,10 @@
+type partialOperationContextFunc = option(UrqlTypes.partialOperationContext) => unit;
```

Comment:
❤️Thanks for generating a `.rei` file! I've always had trouble w/ this due to the CLI mucking up the `[@bs]` attributes. Did you hand write this or did the CLI generate it?

Reply:
Wrote it by hand :)